ReactCommon/yoga/yoga/BitUtils.h

```diff
@@ -45,8 +45,8 @@ void setEnumData(uint32_t& flags, size_t index, int newValue) {
 
 template <typename Enum>
 void setEnumData(uint8_t& flags, size_t index, int newValue) {
-  flags = (flags & ~mask(bitWidthFn<Enum>(), index)) |
-      ((newValue << index) & (mask(bitWidthFn<Enum>(), index)));
+  flags = (flags & ~static_cast<uint8_t>(mask(bitWidthFn<Enum>(), index))) |
+      ((newValue << index) & (static_cast<uint8_t>(mask(bitWidthFn<Enum>(), index))));
```

Comment:
could be written more generically (if you have enums of other widths to share this code with) as:
```suggestion
  flags = (flags & ~static_cast<decltype(flags)>(mask(bitWidthFn<Enum>(), index))) |
      ((newValue << index) & (static_cast<decltype(flags)>(mask(bitWidthFn<Enum>(), index))));
```

Reply:
I'm going to skip this one for now.  The type of the enum is already specified in the parameter, and the decltype is a little more complex that you have here. It would have to be `decltype(std::remove_reference<flags>::type)` which would also add a dependency on `type_traits`.